src/renderer.rs

```diff
@@ -3,10 +3,14 @@ compile_error!("This module should not be included when compiling to wasm");
 
 use std::sync::mpsc::{self, TryRecvError};
 use std::thread;
+use std::path;
 
 use piston_window::{clear, context, line, polygon, AdvancedWindow, Event as PistonEvent, G2d,
     Input, PistonWindow, WindowSettings};
 
+use svg;
+use svg::node::element::{Line as SvgLine, Polygon as SvgPolygon, Rectangle as SvgRect};
```

Comment:
Good use of renaming! It's definitely tough when lots of libraries use the same terms :sweat_smile: